pkg/migrations/op_create_index.go

```diff
@@ -34,9 +34,33 @@ func (o *OpCreateIndex) Start(ctx context.Context, conn db.DB, latestSchema stri
 		stmt += fmt.Sprintf(" USING %s", string(o.Method))
 	}
 
-	stmt += fmt.Sprintf(" (%s)", strings.Join(
-		quoteColumnNames(table.PhysicalColumnNamesFor(o.Columns...)), ", "),
-	)
+	if len(o.Columns) != 0 {
```

Comment:
Could we add comments to the contents of this block? Just one line above each block saying what each section is responsible for deparsing.